.github/workflows/ci.yml

```diff
@@ -2,27 +2,36 @@
 name: CI
 
 on:
-  - push
   - pull_request
 
 permissions:
   contents: read
 
 jobs:
   rspec:
-    runs-on: ubuntu-20.04
+    name: ${{ format('rspec ({0})', matrix.ruby) }}
```

Comment:
I don't like the name, but we need it to match with the _required_ workflows.

Does anyone have an access to change them? Otherwise, we have to support outdated ones for ages 😕